frontend/src/store/lib/deck-validation.ts

```diff
@@ -91,50 +94,72 @@ export type ForbiddenCardError = {
   type: "FORBIDDEN";
   details: {
     code: string;
+    real_name: string;
```

Comment:
I'm not sure we want to do this here. It should be the job of the UI components to resolve the code to a display string.